src/main.cpp

```diff
@@ -1314,12 +1314,27 @@ bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex)
 
 CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
 {
-    int halvings = nHeight / consensusParams.nSubsidyHalvingInterval;
+    CAmount nSubsidy = 50 * COIN;
+
+    // Mining slow start
+    if (nHeight < consensusParams.nSubsidySlowStartInterval / 2) {
+        // Subsidy is ramped up linearly
+        nSubsidy /= consensusParams.nSubsidySlowStartInterval;
```

Comment:
Let's document somewhere that ``nSubsidySlowStartInterval`` needs to evenly divide ``nSubsidy``. Bonus points for somehow causing compilation to fail if this invariant is violated.